src/lib/validators.ts

```diff
@@ -26,6 +27,7 @@ import { validatorsPaths } from './validation/validators-paths';
 const schemas: any[] = [commonSchema, eventsChema, functionsSchema, retriesSchema, workflowSchema];
 const strict: boolean = false;
 const ajv = new Ajv({ schemas, strict });
+addFormats(ajv, ['uri']);
```

Comment:
I'm not sure I understand the suggestion. Instead of using the uri validation regex of ajv-formats, you suggest to add our own validation that basically accept everything but does away with the warnings, is that so?

Reply:
yes it is. as you said hopping [there](https://github.com/serverlessworkflow/sdk-typescript/issues/101#issuecomment-843014234) is no side effects

Reply:
Ok, I just added a custom validator instead that allow anything as 'uri'